forumMessages/tests.py

```diff
@@ -1,2 +1,21 @@
-# from django.test import TestCase
-# Remove comment when implementing tests
+import pytest
+from forumMessages.models import Message, Category
+from django.db.models.query import QuerySet
+
+
+@pytest.mark.django_db
+class TestMessages:
+
+    def test_addCategory(cls):
+        Category.addCategory("TestCat")
+        testCat = Category.objects.get(name="TestCat")
+        assert testCat.name == "TestCat"
+
+    def test_getCategories(cls):
+        allCat = Category.getCategories()
+        assert isinstance(allCat, QuerySet)  # Check if the list of categories is OK after adding a test category
+        assert all(isinstance(cat, Category) for cat in Category)  # Check if all the instances in Category table
```

Comment:
I'm missing assertion of the values inside the object,
what you asserted until here is pure Django behavior